RPIPs/RPIP-vop.md

```diff
@@ -0,0 +1,309 @@
+* Votes and discussions can be labelled explicitly according to the process they are using. This makes it easier for participants to quickly slot items into the appropriate mental boxes. An RPIP is important, because it represents a change to the status quo. A GMC election is also important, but it doesn't represent a change to how Rocket Pool works. A grants appeal is potentially unimportant to a less-engaged participant, and can be more safely ignored. 
+* The outcomes of processes can be recorded in a clear, consistent and accessible manner. Again, the goal is to minimize the attention and mental effort required by potential participants to find information they are looking for.
+
+**Extensibility**
+
+Adoption of a VOP-like framework helps to structure the processes of the Rocket Pool DAO in such a way that is easily extensible in future improvement proposals. The motivation here is to encourage organization, transparency and accessibility in such a way that it becomes self-sustaining. If there is a clear existing structure in use, then it encourages its use in future RPIPs.
+
+## Specification
+$TODO Remove
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+### VOP Definition
+This section describes what a VOP is, and when to use the VOP framework.
+
+* A Voted Operational Process (VOP) is a process involving a vote through which an operational decision is made.
+  * Some operational decisions within the Rocket Pool DAOs MAY NOT require a vote, and therefore are not included in the VOP framework.
+* A VOP MUST NOT be defined to cover decisions that are improvements to the Rocket Pool protocol, Rocket Pool DAOs, or anything that should otherwise require use of the RPIP process.
+* The dividing line between which decisions should use the RPIP process versus the VOP framework may be unavoidably blurry. The following general recommendations are made:
+  * If a proposal can reasonably predicted to be unique, it is RECOMMENDED to use the RPIP process.
+  * If a proposal is the first of its type or class, and it is not anticipated to repeat frequently or regularly it is RECOMMENDED to use the RPIP process.
+  * If a proposal is not the first of its type or class, or if it is anticipated to repeat frequently or regularly it is RECOMMENDED to use the VOP framework. 
+* The VOP framework SHOULD be used for anything that fits the definition of a voted operational process. 
+  * If an RPIP specifies a process that could reasonably fit within the VOP framework, without adhering to the VOP framework, it MUST include detailed rationale as to why adhering to the VOP framework is not preferable.
```

Comment:
Should the opposite be stated as well? Seems like there would equally be times where a VOP could be an RPIP but is selected for a VOP due to one of the reasons mentioned on the motivation section of this doc

Reply:
Misunderstood this I think, I'll try to clarify.